src/errors.h

```diff
@@ -10,6 +10,7 @@
 #define ERRCODE_IO_QUERY_ERRORS MAKE_SQLSTATE('I','O','0','0','0')
 #define ERRCODE_IO_HYPERTABLE_NOT_EXIST MAKE_SQLSTATE('I','O','0','0','1')
 #define ERRCODE_IO_DIMENSION_NOT_EXIST MAKE_SQLSTATE('I','O','0','0','2')
+#define ERRCODE_IO_HISTOGRAM_MAX_LESS_MIN MAKE_SQLSTATE('I','O','0','0','3')
```

Comment:
Do we need a new error code for this? Seem extremely specific. Won't a `ERRCODE_INVALID_PARAMETER_VALUE` suffice?